.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,9 @@
 ### Changelog category (leave one):
-- New Feature
-- Improvement
 - Bug Fix (user-visible misbehaviour in official stable or prestable release)
-- Performance Improvement
-- Backward Incompatible Change
-- Build/Testing/Packaging Improvement
-- Documentation (changelog entry is not required)
-- Not for changelog (changelog entry is not required)
 
 
 ### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
-...
+Fixed the /common/examples/cow_compositions.cpp, it will not clone the inner column when there is no need
```

Comment:
Wrong modification of the template.

Reply:
> Wrong modification of the template.

so sorry， i am newer for clickhouse，when i use the common/examples/cow_composition.cpp， i found that the inner （wrapped）column would be cloned when there is no need，and fixed it，so what should i do，how can i modify the template

Reply:
> Wrong modification of the template.

the bug is presented in:
https://github.com/ClickHouse/ClickHouse/issues/36449#issue-1209576330

Reply:
@zombee0 just leave https://github.com/ClickHouse/ClickHouse/pull/36453/commits/efae3dce1b6088f79c26f253066c3391f572e83c in this PR, you don't need to edit `PULL_REQUEST_TEMPLATE.md` at all, just set correct description in this PR. Or you can enable edit access for this PR ([see](https://docs.github.com/en/enterprise-server@3.4/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests)) and I'll push directly to your branch.

Reply:
I had "Allow edits by maintainers"

Reply:
and, I sync the ClickHouse master, and rollback to efae3dc, is this ok?